site/docs/output_directories.md

```diff
@@ -26,9 +26,9 @@ The solution that's currently implemented:
 
 * Bazel must be invoked from a directory containing a WORKSPACE file. It reports
   an error if it is not. We call this the _workspace directory_.
-* The _outputRoot_ directory is ~/.cache/bazel. (Unless `$TEST_TMPDIR` is
-  set, as in a test of bazel itself, in which case this directory is used
-  instead.)
+* The _outputRoot_ directory is ~/.cache/bazel on Linux, and /private/var/tmp
```

Comment:
Why don't we take the opportunity to improve the readability.  How about

The _outputRoot_ directory defaults to ~/.cache/bazel on Linux, and /private/var/tmp on macOS. If the environment variable `$TEST_TMPDIR` is set, as in a test of bazel itself, then that value overrides the default.

@laszlocsomor Do you know, offhand what we have for Windows. We might as well add that here too, while fixing it.

Reply:
On Windows, we use `%HOME%` if set, otherwise `%USERPROFILE%` if set, otherwise call `SHGetKnownFolderPath` with `FOLDERID_Profile`.

https://github.com/bazelbuild/bazel/blob/22ca5ee2d2805300393f71ae95bb1e5a9f5a168b/src/main/cpp/workspace_layout.cc#L33-L35

https://github.com/bazelbuild/bazel/blob/22ca5ee2d2805300393f71ae95bb1e5a9f5a168b/src/main/cpp/blaze_util_windows.cc#L395-L407

https://github.com/bazelbuild/bazel/blob/22ca5ee2d2805300393f71ae95bb1e5a9f5a168b/src/main/cpp/blaze_util_windows.cc#L409-L440

Reply:
>Why don't we take the opportunity to improve the readability. How about

Sure, why not, took your suggestion and added in the Windows info.

Reply:
>Why don't we take the opportunity to improve the readability. How about

Sure, took your suggestion and added in the Windows info (which I wanted to add but couldn't find, thanks for the info @laszlocsomor ).